### PR TITLE
e2e test case for favorites

### PR DESCRIPTION
**Pull request by lovedeep92**
**JIRA:**
See ticket here - https://rackspace.atlassian.net/browse/MNRVA-868

**Description:**

1) created e2e test case for favourite on button click.
2) Fixed two e2e failed test cases

**Testing:**

`npm run test:e2e`

 ## Screenshots:
(if applicable)

### CODE REVIEW

File: e2e/src/features/visualize/visualizeview/favorites.e2e-spec.ts [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+  nav = new navigations();
+  nav.navigateToVisualize();
+  browser.sleep(1000);
+  favPage=new VisualizeViewPage();
+  });
+
+  it("Should verify the query parameter for favorites after click on favorites button", async () => {
+  favPage.favLink.click();
+  let url = browser.getCurrentUrl();
+  expect(url).toContain('http://dev.i.rax.io:4200/intelligence/visualize?');
```
Comment (FarajiA) [inline e2e/src/features/visualize/visualizeview/favorites.e2e-spec.ts:28]: You may want to leave the hostname _(dev.i.rax.io:4200)_ out of the test. This is always subject to change.

File: e2e/src/pages/visualizeViewPage.ts [hunk 2]
```diff
@@ -28,4 +28,5 @@ export class VisualizeViewPage {
     startTimeMIN = element(by.css("label.owl-dt-timer-content:nth-child(3) > input:nth-child(1)"));
     csvCogMenu = element(by.id(`connLimitExceeded_connLimitExceeded1`));
     csvExportlnk = element(by.id("connLimitExceeded_connLimitExceeded"));
+    favLink = element(by.xpath("/html/body/app-root/div/div/main/hx-panel/hx-panelbody/app-visualize/div[2]/div[1]/div[5]/div/hx-tab"));
```
Comment (FarajiA) [inline e2e/src/pages/visualizeViewPage.ts:31]: This looks to work, that's fine. If you can locate this path starting at `app-visualize` that would be more error proof in the future. If design elements change outside of the feature this test will break.

**Comment by FarajiA** [comment]
I'll also have to check out the failed build. This has to do with the Chrome version being used.

File: e2e/src/features/visualize/visualizeview/favorites.e2e-spec.ts (hunk 1 above)
Comment (lovedeep92) [reply e2e/src/features/visualize/visualizeview/favorites.e2e-spec.ts:28]: done!!

File: e2e/src/pages/visualizeViewPage.ts (hunk 2 above)
Comment (lovedeep92) [reply e2e/src/pages/visualizeViewPage.ts]: done!!

**Review by FarajiA — APPROVED** [review]

**Pull request merged**